concordium-std/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
   to be consistent with the Write implementation for ContractState.
 - Use little-endian encoding for sender contract addresses in receive contexts. This
   reverts the change in concordium-std 0.4.1.
+- Change the `receive_name` parameter of `HasActions::send` to use `ReceiveName`
+  instead of `str`.
+- Add `simple_send`, a wrapper for `HasActions::send`, which automatically
```

Comment:
I think we need a better name than `simple_send` so as to not confuse people (there is `simple_transfer` already)